frontend/src/theme/index.tsx

```diff
@@ -13,6 +13,12 @@ const customTheme = extendTheme({
       fontFamily: "Open Sans",
       fontSize: "1vw",
     },
+    h4: {
+      color: "#fff", margin: "30px",
+      fontFamily: "Coustard",
+      fontSize: "20px";
```

Comment:
can you put it in terms of viewport/% so that it can change its size accordingly to the user's screensize?